koku/api/resource_types/aws_ec2_compute_instances/view.py

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright 2024 Red Hat Inc.
+# SPDX-License-Identifier: Apache-2.0
+#
+"""View for AWS accounts."""
+from django.conf import settings
+from django.db.models import F
+from django.db.models.functions import Coalesce
+from django.utils.decorators import method_decorator
+from django.views.decorators.vary import vary_on_headers
+from rest_framework import filters
+from rest_framework import generics
+from rest_framework import status
+from rest_framework.response import Response
+
+from api.common import CACHE_RH_IDENTITY_HEADER
+from api.common.pagination import ResourceTypeViewPaginator
+from api.common.permissions.aws_access import AwsAccessPermission
+from api.resource_types.serializers import ResourceTypeSerializer
+from reporting.provider.aws.models import AWSCostEntryLineItemSummaryByEC2Compute
+
+
+class AWSEC2ComputeInstanceView(generics.ListAPIView):
+    """API GET list view for AWS EC2 compute instances."""
+
+    queryset = (
+        AWSCostEntryLineItemSummaryByEC2Compute.objects.annotate(
+            **(
+                {
+                    "value": F("resource_id"),
+                    "alias": Coalesce(F("instance_name"), "resource_id"),
+                }
+            )
```

Comment:
This seems like an overly complicated way of passing in kwargs. Is there a reason unpacking is used here instead of plain kwargs? `annotate()` [accepts arbitrary args and kwargs](https://docs.djangoproject.com/en/5.0/ref/models/querysets/#django.db.models.query.QuerySet.annotate).
```suggestion
            value=F("resource_id"),
            alias=Coalesce(F("instance_name"), "resource_id"),
```

Reply:
I've update this to passing plain kwargs